listenbrainz/webserver/templates/user/import.html

```diff
@@ -3,45 +3,55 @@
-      by Last.fm, and we cannot use it. Instead, we use the term <strong>listen</strong> for our data.
-    </p>
+        <p>
+            Most users will want to import from Last.fm directly.
+            <br>
+            Fun Fact: The term <strong>scrobble</strong> is a trademarked term
+            by Last.fm, and we cannot use it. Instead, we use the term <strong>listen</strong> for our data.
+        </p>
 
-    <h3>Direct import from Last.fm and Libre.fm</h3>
-      <p>
-        The importer manually steps through your listen history and imports the listens
-        one page at a time. Should it fail for whatever reason, it is safe to restart the import
-        process. Running the import process multiple times <strong>does not</strong> create duplicates in your
-        ListenBrainz listen history.
-      </p>
-      <p>In order for this to work, you must disable the &#34;Hide recent listening information&#34;
-        setting in your Last.fm <a href="https://www.last.fm/settings/privacy">Privacy Settings</a>.
-      </p>
-      <p>
-        Clicking the "Import now!" button will import your profile now without the need to open lastfm.<br/>
-        You need to keep this page open for the tool to work, it might take a while to complete. Though, you can continue doing your work. :)
-      </p>
-      <p>
-        <em>NOTE</em>: Please note that your listen counts may not be accurate for up to 24 hours after
-        you import your data!
-        Also, running the Last.fm importer twice will not cause duplicates in your listen history.
-      </p>
-      <div class="row">
-      <div class="well col-sm-8 col-sm-offset-2 col-md-6 col-md-offset-3">
-        <div id="react-container">
+        <h3>Direct import from Last.fm and Libre.fm</h3>
+        <p>
+            The importer manually steps through your listen history and imports the listens
+            one page at a time. Should it fail for whatever reason, it is safe to restart the import
+            process. Running the import process multiple times <strong>does not</strong> create duplicates in your
+            ListenBrainz listen history.
+        </p>
+        <p>In order for this to work, you must disable the &#34;Hide recent listening information&#34;
+            setting in your Last.fm <a href="https://www.last.fm/settings/privacy">Privacy Settings</a>.
+        </p>
+        <p>
+            Clicking the "Import now!" button will import your profile now without the need to open lastfm.<br/>
+            You need to keep this page open for the tool to work, it might take a while to complete. Though, you can
+            continue doing your work. :)
+        </p>
+        <p>
+            <em>NOTE</em>: Please note that your listen counts may not be accurate for up to 24 hours after
+            you import your data!
+            Also, running the Last.fm importer twice will not cause duplicates in your listen history.
+        </p>
+        <div class="row">
```

Comment:
I would wrap this whole row with the same `{% if user_has_email %}` to avoid the empty container

Reply:
Done.